src/kirby/components/toolbar/toolbar.component.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  ContentChild,
+  TemplateRef,
+  HostBinding,
+  Output,
+  EventEmitter,
+} from '@angular/core';
+
+import { ToolbarEndElementDirective, ToolbarSecondaryElementDirective } from './toolbar.directive';
+
+@Component({
+  selector: 'kirby-toolbar',
+  templateUrl: './toolbar.component.html',
+  styleUrls: ['./toolbar.component.scss'],
+})
+export class ToolbarComponent implements OnInit {
+  @Input() title: string;
+  @Input() hideBackButton: boolean;
+
+  @Output() back: EventEmitter<null> = new EventEmitter<null>();
+
+  @ContentChild(ToolbarEndElementDirective, { read: TemplateRef }) endElement;
```

Comment:
Still need answer on if these two templates are needed? Could a `ng-content` not be a better solution?

Reply:
for some reason, I can't get ng-content to work the intended way

Reply:
NVM, got it to work